pkg/project/resource_project_repo.go

```diff
@@ -127,10 +128,19 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 	return readRepos(projectKey, m)
 }
 
+var retry5xxRange = func(response *resty.Response, err error) bool {
+	return response.StatusCode() >= 500 && response.StatusCode() <= 599
```

Comment:
First, you can't have a value > 599 - that's just BS and should error

Second, this isn't specific enough. 5XX codes are not universally retryable.

Reply:
if I remember correctly I saw different HTTP Status code other than 502. I can make it specific to 502

Reply:
comparison here is 500 <= x <=599. Here I am not comparing anything >599.